experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/EchoBot.runtimeconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp2.1",
+    "framework": {
+      "name": "Microsoft.AspNetCore.App",
+      "version": "2.1.1"
+    },
+    "configProperties": {
+      "System.GC.Server": true
```

Comment:
Is this something we want? I don't believe overriding the defaults value here is a good idea. We should just leave the ASP.Net defaults "as-is".